selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/ImportMavenProjectFromGitTest.java

```diff
@@ -61,6 +66,7 @@ public void checkAbilityImportMavenProjectTest() throws ExecutionException, Inte
 
     workspaces.clickOnAddWorkspaceBtn();
     newWorkspace.waitToolbar();
+    newWorkspace.clickOnAllStacksTab();
```

Comment:
please, leave comment about why we can't rely on Quick Start stack list.

Reply:
The **Java** stack, which we use for creating test workspaces, not exist **Quick Start** stack list on **OSIO**.